### ISSUE TITLE
Error when moving certain window type to workspace.

### ISSUE THREAD

**Issue by kyx0r** [imawizard/MiguruWM#18]
Hello I get this error on certain types of borderless windows.

```
Error: This value of type "Integer" has no property named "workspace".

	---- C:\msys64\home\User\progs\MiguruWM\lib\miguru\miguru.ahk
	529: {
	530: window := this._managed.Get(hwnd, 0)
▶	531: next := window.workspace.Remove(hwnd)
	532: ws.AddIfNew(hwnd)
	533: ws.ActiveWindow := hwnd

The current thread will exit.

Call stack:
C:\msys64\home\User\progs\MiguruWM\lib\miguru\miguru.ahk (531) : [MiguruWM.Prototype._onRequest] next := window.workspace.Remove(hwnd)
C:\msys64\home\User\progs\MiguruWM\lib\miguru\events.ahk (151) : [WMEvents.Prototype._onMessage] ret := this._onRequest(req)
C:\msys64\home\User\progs\MiguruWM\lib\miguru\events.ahk (84) : [Func.Prototype.Call] Return  fn.Call(ObjFromPtrAddRef(self), args*)
C:\msys64\home\User\progs\MiguruWM\lib\miguru\events.ahk (84) : [] Return  fn.Call(ObjFromPtrAddRef(self), args*)
> OnMessage

```

**Comment by imawizard** [comment]
Hey Kyryl,
thanks for the report.

You're getting the error when trying to send such borderless windows to other desktops/workspaces, right? Can you name an example for a non-working window?

Fixing the error is trivial with an early return `if !window { return }`, I'm not sure whether that'd make sending such windows work as expected, though.
I'll try to reproduce the behavior and will push the `return`-fix nonetheless.

**Edit**:
Ah right, borderless windows are not managed, so they're not tracked as either tiled or floating, however, sending to another desktop/workspaces uses COM functions so it'll work on unmanaged windows, too, so the fix here is just the `if` mentioned above.

Sending to another monitor only works for managed windows, though. So, if you also want to send it to another monitor, we'd need to think over the use-case again.

**Comment by kyx0r** [comment]
At the moment I don't use multiple monitors, so I can't tell how it behaves there. The fix would be highly appreciated! Waiting for you to update the ahk-v2 branch so I can start using it!

**Comment by imawizard** [comment]
Should be fixed with b04e7f35f0b53596aa52f2b1f8f9147113aaa73e
